src/controllers.rs

```diff
@@ -0,0 +1,60 @@
+extern crate sysfs_gpio;
```

Comment:
`extern`s should go in the crate root, in your case `lib.rs`. You'll only ever need to `extern` a given crate once for your whole crate.